src/Components/Aspire.Oracle.EntityFrameworkCore/README.md

```diff
@@ -0,0 +1,113 @@
+# Aspire.Oracle.EntityFrameworkCore library
+
+Registers [EntityFrameworkCore](https://learn.microsoft.com/ef/core/) [DbContext](https://learn.microsoft.com/dotnet/api/microsoft.entityframeworkcore.dbcontext) service for connecting Oracle database. Enables connection pooling, health check, logging and telemetry.
+
+## Getting started
+
+### Prerequisites
+
+- Oracle database and connection string for accessing the database.
+
+### Install the package
+
+Install the .NET Aspire Oracle EntityFrameworkCore library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Oracle.EntityFrameworkCore
+```
+
+## Usage example
+
+In the _Program.cs_ file of your project, call the `AddOracleDatabaseDbContext` extension method to register a `DbContext` for use via the dependency injection container. The method takes a connection name parameter.
+
+```csharp
+builder.AddOracleDatabaseDbContext<MyDbContext>("orcl");
+```
+
+You can then retrieve the `MyDbContext` instance using dependency injection. For example, to retrieve the context from a Web API controller:
+
+```csharp
+private readonly MyDbContext _context;
+
+public ProductsController(MyDbContext context)
+{
+    _context = context;
+}
+```
+
+## Configuration
+
+The .NET Aspire Oracle EntityFrameworkCore component provides multiple options to configure the database connection based on the requirements and conventions of your project.
+
+### Use a connection string
+
+When using a connection string from the `ConnectionStrings` configuration section, you can provide the name of the connection string when calling `builder.AddOracleDatabaseDbContext<TContext>()`:
+
+```csharp
+builder.AddOracleDatabaseDbContext<MyDbContext>("myConnection");
+```
+
+And then the connection string will be retrieved from the `ConnectionStrings` configuration section:
+
+```json
+{
+  "ConnectionStrings": {
+    "myConnection": "Data Source=TORCL;User Id=myUsername;Password=myPassword;"
+  }
+}
+```
+
+See the [ODP.NET documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/odpnt/#Oracle%C2%AE-Data-Provider-for-.NET) for more information on how to format this connection string.
```

Comment:
For the documentation link, Is there a way to link to a non-English page, and then it redirects to the correct locale for a given user? For links to learn.microsoft.com, we don't include the locale `en-us`.

Reply:
I think I found the correct link this time